common/scala/src/main/scala/org/apache/openwhisk/http/BasicRasService.scala

```diff
@@ -39,4 +40,14 @@ trait BasicRasService extends BasicHttpService {
   val ping = path("ping") {
     get { complete("pong") }
   }
+  val ready = path("ready") {
+    get {
+      if (readyState) {
+        complete("ok")
+      } else {
+        logging.warn(this, "not ready...")
```

Comment:
can this just be a debug? I think ready route returning 503 should be self explanatory